docs/src/tutorials/conversions_views.md

```diff
@@ -507,3 +507,33 @@ Base.ReshapedArray{RGB{Normed{UInt8,8}},3,Base.ReinterpretArray{RGB{Normed{UInt8
 While there is little or no performance cost to making use of
 JuliaImage's convenient views, sometimes the types can get
 complicated!
+
+## Preserving colorant type while conversion
+
+While using Float32, Float64 to convert the raw storage type of an image 
+the color space information is lost. Replacaing Float32, Float64 with their lower-case
+forms preserves the colorant type.
+
+Take an example of converting a Gray image to Float64. Simply broadcating Float64
+will promote the storage to Float64 but the color space information is lost.
+However, when we use float64. The colorant type inforamtaion is preserved.
+
+```jldoctest; setup = :(using TestImages, ColorTypes; img = testimage("cameraman");)
+julia> img |> eltype
+Gray{Normed{UInt8,8}}
+
+julia> Float64.(img) |> eltype
+Float64
+
+julia> float64.(img) |> eltype
+Gray{Float64}
+```
+|  Storage Type 	|  Alternative 	|
+|---	|---	|
+| Float32  	|  float32 	|
```

Comment:
Let's quote these, too. And it would be better to quote the lower-cased ones with ref-links:

```
[`float32`](@ref)
```